src/test/IexTest/CMakeLists.txt

```diff
@@ -13,4 +13,4 @@ set_target_properties(IexTest PROPERTIES
 if(WIN32 AND BUILD_SHARED_LIBS)
   target_compile_definitions(IexTest PRIVATE OPENEXR_DLL)
 endif()
-add_test(NAME OpenEXR.Iex COMMAND $<TARGET_FILE:IexTest>)
+add_test(NAME OpenEXR.Iex COMMAND ${CMAKE_CROSSCOMPILING_EMULATOR} $<TARGET_FILE:IexTest>)
```

Comment:
I haven't seen this feature before ~ could you add a brief line of documentation here and where it appears elsewhere as to why this is here for future maintainers? For example, "to support cross compiling win32 from mingw" or whatever makes sense.